tests/jenkins/TestDataPrepperReleaseArtifacts.groovy

```diff
@@ -61,20 +62,33 @@ class TestDataPrepperReleaseArtifacts extends BuildPipelineTest {
     void 'downloads from the correct URLs'() {
         runScript('jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile')
 
-        def curlCommands = helper.callStack.findAll { call ->
+        def shCommands = helper.callStack.findAll { call ->
             call.methodName == 'sh'
         }.collect { call ->
             callArgsToString(call)
-        }.findAll { shCommand ->
-            shCommand.contains('curl')
         }
 
-        assertThat(curlCommands, hasItem(
+        def archiveCommands = shCommands.findAll { shCommand ->
```

Comment:
I think this test could be split into two:
* `downloads archives from the correct URLs`
* `downloads Maven artifacts from the correct URLs`


You might find it useful to extract the command to get `curl` commands into a function.